docs/audits/viewport-ad-density.md

```diff
@@ -1,10 +1,12 @@
-# Reduce ad density in initial viewport
+# Reduce ad density
 
 ## Overview
 
-This audit ensures that visible ads take up no more than 30% of the initial
-viewport. An ad density greater than this has been found to negatively impact
-user experience[^1].
+This audit ensures that ads take up no more than 30% of the page vertically. On
```

Comment:
I think it probably makes more sense to move this into the "more information" section, since these are implementation details. Recommended wording:

On mobile devices, a single column layout is assumed, and all ads on the page are included in the ad density calculation. On desktop, the page is divided into multiple vertical slices to account for multi-column layouts. The density of the ads included in each slice are then calculated, to ensure no individual slice exceeds the maximum recommended threshold.

Reply:
I agree with moving this paragraph to **more information**. One minor change: I would say something like _we assume a single-column layout_ or _this audit assumes a single-column layout_, since _is assumed_ is a little ambiguous (assumed by whom?).

Reply:
Done. Went with the active voice